governance/paper-process.md

```diff
@@ -0,0 +1,235 @@
+content:
+
+* Who do we expect to read the paper?
+* What positions do they hold?
+* Why are we writing this paper?
+* What problem are we intending to resolve?
+* What assumptions are we making about the audience or the expected content of
+  the paper?
+  * Note - these are documented in the "Introduction > Assumptions"
+      section of the paper
+* What are our Goals and Non-Goals?
+  * Note - these are documented in the "Introduction > Scope > Goals" and
+      "Introduction > Scope > Non-Goals"
+* Is the scope in the issue still representative of what we intend to cover?
+
+### Tasking assignment
+
+Contributors should highlight and assign themselves to various areas of the
+outline.  This allows contributors to focus on a few areas and identify others
+where more contributors is needed.  If using a google doc for the paper,
+assignments may be done through the "assign to me" feature after highlight the
+area or by using the "+foo@bar.com" comment feature for the assignee's email.
+Once task assignment is complete, contributors can begin content rough-in.
+
+### Content rough-in
+
+Assignees can provide content rough-in in a variety of ways.  They may provide
+raw content in the form of phrases, disjointed paragraphs, bullets on the topic,
+or draft content in the form of completed paragraphs.  If leveraging draft
+content, placing this content in quotes helps other reviewers know it is
+'near-complete' and it should be reviewed as a holistic section.  
+
+#### Opinions
+
+There may be situations where multiple differing views on the subject matter or
+controls implementation are expressed.  Authors should make every effort to
+discover the industry standard or practice and document it. If none exists
+and the suggested content cannot be collaboratively resolved, it is the duty
+of the authors to capture the different views with references, benefits, and
+drawbacks or contingencies in order to provide the reader with enough
+information to reach their own conclusion.
+
+#### References
+
+It is important that any references used to generate paper content be cited
+appropriately through footnotes and within the references section.  This
+permits the reader to not only dig deep into a specific reference area but also
+provides source content and is a matter of good practice.  
+
+### Collaborative
+
+Contributors now review all content roughed-in and transition raw content into
+draft content.  They may begin commenting on sections, expanding ideas, refining
+arguments, and providing citations.  Contributors are encouraged to discuss
+content specific items in the doc through comments whereas larger themes and
+writing decisions should be discussed in the corresponding slack channel.  This
+keeps the doc clean as well as documents thematic decisions in the primary
+collaboration & coordination channel.
+
+### Executive summary and content wrap-up
+
+If the paper exceeds five (5) pages, an Executive summary commensurate with the
```

Comment:
Given that markdown and blogs are the prevalent forms of CNCF publications, page limit has very little meaning; it only adds confusion.

Reply:
This md exclusively talks about publishing papers, which are published as pdf. Some of these papers go 20+ pages.

Reply:
Let's remove anything overly prescriptive.